src/services/WorkspaceFlowRunsApi.ts

```diff
@@ -75,8 +75,12 @@ export class WorkspaceFlowRunsApi extends WorkspaceApi {
   }
 
   public async getFlowRunInputDescription(id: string, keyset: FlowRunInputKeyset): Promise <string | null> {
-    const { data } = await this.get<string | null>(`/${id}/input/${keyset.description}`)
-    return data
+    try {
+      const { data } = await this.get<string | null>(`/${id}/input/${keyset.description}`)
+      return data
+    } catch (error) {
+      return null
+    }
```

Comment:
@abrookins Why the try catch here?

Reply:
Because otherwise the 404 exception crashes the modal.

Reply:
Ah, I should have mentioned. The form is currently broken without this -- I had to fix it to make my changes.

Reply:
Just updated the PR description to reflect this context

Reply:
oh I see because you're not using a subscription in the modal. I'd recommend not swallowing errors here and using a subscription in the modal.

```
const { response: inputDescription } = await useSubscription(api.flowRuns.getFlowRunInputDescription, props.flowRunId).promise()
```

Or at least `console.error(error)` in your catch so that any errors still show up in the devtools console.